### PR TITLE
CASMPET-5384: added pie chart plugin to Grafana

### PR DESCRIPTION
**Pull request by rambabubolla**
Scope and Summary:
The existing Grafana docker image doesn't have Piechart plugin in it.
Downloaded the Piechart plugin and installing it into the path /var/lib/grafana/plugins/
Jira ticket: CASMPET-5384

Testing: Logged into the image and found that the piechart plugin is available in the path /var/lib/grafana/plugins/

Changes: Added zip file. Modified the Dockerimage. And bumped the version.

### CODE REVIEW

File: .github/workflows/docker.io.grafana.grafana.7.0.3.yaml [hunk 1]
```diff
@@ -56,7 +56,7 @@ jobs:
         uses: Cray-HPE/github-actions/build-sign-scan@main
         with:
           # Only push on main builds
-          docker_push: ${{ github.ref == 'refs/heads/main' }}
+          docker_push: true
```
Comment (mtupitsyn) [inline .github/workflows/docker.io.grafana.grafana.7.0.3.yaml:59]: Please remove this change - I assume this was done temporarily to push to registry dev branch. Genrally, we don;t push from dev branches, only from master.

**Comment by mtupitsyn** [comment]
@rambabubolla You may want to add someone from PET team as reviewers. I don't think Ludwig and Wilton are doing reviews in this repository anymore.

File: docker.io/grafana/grafana/7.0.3/Dockerfile [hunk 2]
```diff
@@ -27,6 +27,12 @@ USER root
 
 RUN apk add --upgrade apk-tools &&  \
     apk update && apk -U upgrade && \
+    apk add zip && \
+    apk add unzip && \
     rm -rf /var/cache/apk/*
 
+COPY ./plugins/grafana-piechart-panel.zip /var/lib/grafana/plugins
```
Comment (mtupitsyn) [inline docker.io/grafana/grafana/7.0.3/Dockerfile:34]: Any chance grafana-piechart-panel.zip can be downloaded from trusted external location during build? I briefly tried this, seems to be achieving the same result:

    ~$ docker run -it --rm --entrypoint bash grafana/grafana:7.0.3
    bash-5.0$ grafana-cli plugins install grafana-piechart-panel
    installing grafana-piechart-panel @ 1.6.2
    from: https://grafana.com/api/plugins/grafana-piechart-panel/versions/1.6.2/download
    into: /var/lib/grafana/plugins
    
    ✔ Installed grafana-piechart-panel successfully
    
    Restart grafana after installing plugins . <service grafana-server restart>

**Comment by rambabubolla** [comment]
> @rambabubolla You may want to add someone from PET team as reviewers. I don't think Ludwig and Wilton are doing reviews in this repository anymore.

Sure will add brant

**Pull request closed without merging**